README.md

```diff
@@ -247,3 +247,15 @@ We built our snapshot testing setup with `SnapshotTestHelper` to streamline UI c
 - **Comparison**: We capture snapshots of the UI and compare them to reference images to spot any unintended changes.
 
 More details [here](SNAPSHOT_TESTING_WIKI.md)
+
+## Ecosia framework
```

Comment:
Maybe we should move our Ecosia specific README sections to the Framework's README?

Reply:
I agree with that 👍 - let me do that so we have  a dedicated place